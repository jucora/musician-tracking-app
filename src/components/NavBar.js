import React from 'react';
import { NavLink } from 'react-router-dom';

const NavBar = () => {
  return (
    <div className="navBar">
      <div className="navBarLinkBox">
        <NavLink
          className="nav-link"
          activeClassName="nav-link-active"
          to="/skillForm"
        >
          Add Skill
        </NavLink>
      </div>
      <div className="navBarLinkBox">
        <NavLink
          className="nav-link"
          activeClassName="nav-link-active"
          to="/track"
        >
          Track.it
        </NavLink>
      </div>
      <div className="navBarLinkBox">
        <NavLink
          className="nav-link"
          activeClassName="nav-link-active"
          exact
          to="/progress"
        >
          Your Progress
        </NavLink>
      </div>
      <div className="navBarLinkBox">
        <NavLink
          className="nav-link"
          activeClassName="nav-link-active"
          exact
          to="/more"
        >
          More
        </NavLink>
      </div>
    </div>
  );
};

export default NavBar;
