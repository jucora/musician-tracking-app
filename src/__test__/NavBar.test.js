import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import NavBar from '../components/NavBar';

import { BrowserRouter } from 'react-router-dom';

afterEach(cleanup);

it('renders', () => {
  const { asFragment } = render(
    <BrowserRouter>
      <NavBar />
    </BrowserRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
