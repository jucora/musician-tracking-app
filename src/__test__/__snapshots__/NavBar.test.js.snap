// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="navBar"
  >
    <div
      class="navBarLinkBox"
    >
      <a
        class="nav-link"
        href="/skillForm"
      >
        <div
          style="width: 100%; text-align: center;"
        >
          <img
            alt="Add Skill"
            src="[object Object]"
            style="width: 25%;"
          />
        </div>
        Add Skill
      </a>
    </div>
    <div
      class="navBarLinkBox"
    >
      <a
        class="nav-link"
        href="/track"
      >
        <div
          style="width: 100%; text-align: center;"
        >
          <img
            alt="Track"
            src="[object Object]"
            style="width: 25%;"
          />
        </div>
        Track.it
      </a>
    </div>
    <div
      class="navBarLinkBox"
    >
      <a
        class="nav-link"
        href="/progress"
      >
        <div
          style="width: 100%; text-align: center;"
        >
          <img
            alt="Progress"
            src="progress.png"
            style="width: 25%;"
          />
        </div>
        Your Progress
      </a>
    </div>
    <div
      class="navBarLinkBox"
    >
      <a
        class="nav-link"
        href="/more"
      >
        <div
          style="width: 100%; text-align: center;"
        >
          <img
            alt="More Option"
            src="[object Object]"
            style="width: 25%;"
          />
        </div>
        More
      </a>
    </div>
  </div>
</DocumentFragment>
`;
